chess_gui.py

```diff
@@ -147,18 +183,66 @@ def main():
                             player_clicks = []
                             valid_moves = []
                         else:
-                            game_state.move_piece((player_clicks[0][0], player_clicks[0][1]),
+                            move_num += 1
+                            for i in range(0, len(game_state.board)):
+                                for j in range(0, len(game_state.board[i])):
+                                    if game_state.board[i][j] != -9:
+                                        logging.info(f"move number {move_num}: {game_state.board[i][j]}")
+                            res = game_state.move_piece((player_clicks[0][0], player_clicks[0][1]),
                                                   (player_clicks[1][0], player_clicks[1][1]), False)
```

Comment:
Try to avoid using 'i' and 'j' if better names are avalible
Rename 'res'

You are using these loops  a few times, consider seperating it to a new function

Reply:
Got fixed